src/SFML/Window/Context.cpp

```diff
@@ -64,6 +64,11 @@ Context::~Context()
     }
 }
 
+////////////////////////////////////////////////////////////
+Context::Context(Context&& other) noexcept = default;
+
+////////////////////////////////////////////////////////////
+Context& Context::operator=(Context&& other) noexcept = default;
```

Comment:
The default implementation is not correct. We need to deactivate the context before destroying it.
I think this would be a correct implementation:
```c++
Context& Context::operator=(Context&& other) noexcept
{
    if (&other == this)
        return *this;

    if (m_context && !setActive(false))
        err() << "Failed to set context as inactive during move assignment" << std::endl;

    m_context = std::move(other.m_context);

    return *this;
}```